src/libs/odrl/rules.js

```diff
@@ -5,7 +5,7 @@ export const RuleTypes = Object.freeze({
     hasParentRule: false,
     subrule: 'duty',
     description: 'ist erlaubt.',
-    descriptionAddition: ['', ''],
```

Comment:
I thought a shorter name would increase readability in RuleItem

Reply:
Also description addition both contain the postfix '**tion**' which doesn't look nice.